### ISSUE TITLE
CenterCrop3D / RandomCrop3D raise _LinAlgError for a crop with a size-1 axis

### ISSUE THREAD

**Issue by ducha-aiki** [kornia/kornia#4705]
## Summary

`CenterCrop3D` and `RandomCrop3D` crash for any crop with a size-1 axis, whichever axis it is. The 2D crops accept the same request.

```python
import torch, kornia.augmentation as K
x = torch.rand(1, 1, 4, 5, 6)
for size in ((1, 3, 4), (2, 1, 4), (2, 3, 1)):
    for cls in (K.CenterCrop3D, K.RandomCrop3D):
        cls(size, p=1.0)(x)
# torch._C._LinAlgError: torch.linalg.solve: (Batch element 0): The solver failed because the input matrix is singular.
#   ... for all six combinations

K.CenterCrop3D((2, 3, 4), p=1.0)(x).shape        # torch.Size([1, 1, 2, 3, 4])  fine
K.CenterCrop((1, 3), p=1.0)(torch.rand(1, 1, 4, 5)).shape   # torch.Size([1, 1, 1, 3])  the 2D class is fine
K.RandomCrop((1, 3), p=1.0)(torch.rand(1, 1, 4, 5)).shape   # torch.Size([1, 1, 1, 3])
```

Measured on `main` @ ff204fb9c, torch 2.14.0, CPU float32.

## Cause

Both classes build the crop from `get_perspective_transform3d(src_box, dst_box)`. A size-1 axis makes the eight `src` (and `dst`) corners coplanar — the inclusive box `[z0, z0 + size - 1]` has zero extent — so the 3D DLT system is rank-deficient and `torch.linalg.solve` raises. A crop is an axis-aligned translation, so no perspective solve is needed to express it.

## Expected

A size-1 axis crops to a single slice, as in 2D (extracting a single slice from a volume is an ordinary request). If that is not going to be supported, the constructor should reject it with a `ValueError` naming the constraint rather than surfacing a linear-algebra error from inside the warp.

Found while reviewing #4634 (batch 6d of #4407); no Convention sentence there covers a size-1 axis, so it is not documented as behaviour.

Posted on behalf of @ducha-aiki by Claude (Fable 5.1).
